02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,54 @@
ETPWVbxyWu0SowbV3/Z/njtbVUccAljEiKt1RX8TEn+umGaOpE2sAKird3dlFh36/TERmEr1DGRjaPV4U7AE9cKMG3jFgVktTRtmPLqrhJTYuOqjuRv19b+mDmE3vskUUQd1Yl4tF5L+npivZK8iZlBJEAHBO5Nh0PfEnUGKcBivLfuRYg/TFzjjAkXkqoahZamnUuoFxURHxfNt7k/XDnv1KZA39mxT2N2WN2F/rcgD0t2xXplcL4ZC5bY2PTHaZEUWkldG6+A9fXC4L2AfV/iVUj0cJp1a34PNJI+p649RUdRWPIvMjgCj4528AHmT5frgRuUD4Xc26Nc3IwW9XSStCGinKotmZ5Nx8lsBt063w3+EBOe8BKkGiEpo6SRREDezWUC7W7mxNu1ziPz6uhqMxqGu6k2IFth4Rcffv3wqk4jroqIZZTUMBp9yquSxB633O+9+vnfENmRleolM0fLkZt1HRcZRr2ese9YS2TSGSlDRLI0oNgVe2kjzv9Nv6CDHVQ1JSSVSV3ZGUbE4g8vqpYX5EUmi5Njc9eh6fLBs4Zm3nRzte5Jv8/wDb5Yp143gaWOfMfdZHlo6mnCuLBNOq4v57EflhqPN5yt2rIV36FXb9ScQsxCseS9kHhGtNz+WFJVBVtphb5hFXE8BjtmKGYKBGNixA/iMMyckqqgycvTuW7nvpHr3w1JLrj/FkZ5NZIuo079Te/X6YbEoUG1ulvPFpALEMZB0roQsbO36YCq4WaXSrM1xtpQm+CkmZelu/QeeJ6F6WGCnYwESvErNKJtJOoX6EHGkd0mTSK37pVrDfkOkSDckW++F01FPUJzLMI721kfpif5dFUheYs8gXcD3hbX87BMS9Pkhny2Opj58NJq0gLU7glrXto88N8iPqIpr0hiZlYHwkAi1/XC4VaMtqVVBUgm24H1w40Uis8gAQKCSWNyfPp/W+BzPqNzILg9bdcQ9ZoEiibRq5MhF7X0Gx+fphBUbBVH2wlqyVpAHlc33I1kbdOmEiVo/7wLqvp2bSb4XFj06wI2tceWHJJjUUyU8xUaGJSQjoLdDgdnDGyKRfazW2x653Cm6/kMPABauDQ4aMgkHr0x2KUr8SsT2scPv5K5Jt0x7UDYbdOvfF6I7eaWMWhcj9kEYaJkJ3jf6H/bDysuoC9+4364UtLPIoZU2+TDE6A2YWXSWkCq3cMD+WG20hzpYsAdiRa+EstiSCN+uODofEv3w8AcUgkXNh3xq3ssAOaSmJ0DDJGKPLbSreCxPbbGTBgAPDcDuDjRODc7gyV46mqpmqI6jLBTskb6DZgvf0GKj0yLGui15m9TW8FZvLmVVleczQGNoJMu0FqbfxMxUDbby877dIjPeHaDh6pp40zM1NWSknI5VtKEXuT06jYeWEtn+U0eU11FkOTyUsldFyZJqipMtk32APqcD55miZ5nq1qQtEpSOPSx1Hwi3bClJGDaGqX2eVNStDR5hLXQ1lXBz1kgo70sGoEhJHJvfbsNr4D4e4Epc5NJoqsz95qWkUyU9FrpqV1uLSSG17lT8OBK/jDKM3ho6nPMnq581padYS8NZy4agLfTzBbV3309fPB+Xe0LK4P+Hqmqy6q97yeLkqkFUEgdLW1aSCdVj5+pxfR0kpl2VVXEXANHQTXC0ueSrU1UUSloYY42Jc2tq3Pqbjr0xEcG0dFUycUNlteJYEy52RquhVnlQDdl8VkN7dje/QWwzHx/T0GVinyKmraesTNHzCKaWoV1Oq4KMoAupQ2t9b4VTcZcPUeZZtV0+Q10AzSlaCaCOqTRGW+Irddh0sPXBiAIpOBsjemyCSr4iqEbOkYU6JRdHBA332UFgNzuelsAJwhDl1LU1fEFdURU65mcsRKKESNJILnWdRAC27bk48vGGViHheIZZWlcivqJqE/GuQ2223jUfS4674umVZ7FmOUVOY5HR1sk1Xmck1RSUlXCk0RIXSZOYpFrg6SBsO98CSAzrjHKFyLPqjJ6WeSZKdUDyuirqYi5sB0FiNjv1xBWMZu1gR0uLg4neOFoaXiFxllZNUrJEstSZqgTtHO19acwbNbbfzvivGUhSHAfc6VJ39cTgBazPpIIW3r0x33j9y5HoMRdySeo+uOkue5ODggCpXDhiqlCRsF+HA6wyFWOoBV7+Z8seDkkAG5x0sZPBGfAvfzwJYMQpcDY2wfFm+YwxpGlWQiDSoKKbDy3GASVFtIO3njmpdVzscMRJHOs0/+43/AOaf+OCMqzvM2zSiV6ssjVEasDGm4LAEdPniEJF/iwXlO+bUFiP/AHUX/eMNCxGoU/CGc1dPFU02XUZimXVGSsI1DtscQldTTZfVvS11HFDKjAOjU8dx+Vvri655PksWT8NNnFJXVDiivH7tIFsLjrfe/pg3O8u53EWY5nmENDNSDL4pYpa8FYqYMSAHjFy7eE9wN8TxMGn+TNeKhljZXUf2YpNo1LGSGNSp5sYGkhQehb7/AEFby7IcyzHLa7MaOlMlHQIWqZdajRtfoTcm3yxrWc8OZRmuZ8P00CwR02bU+qoakjaJH0Sxm6Kd1ve32xEZdWRSZf7Q6CDJqHLo6KikjQU8JR9IZgA5J8R2vc74vDWCxGVMSp2J+uOEKSpa7W8wOmGy1/XHdWEWOKL309BjwYjY44shWMhep64QRfcnAAu433645bCNJAvtbCgdsACrECwtjt287emPDHT0wgEX+WE9eowvthIwAe0i1rYKysrFmVEzEKoqYySegAYYGGOjrhgapS8Z8Q0VNFTUuZwpDEoSNbQnSB03IwLBxRnsFdPWDMkkmqFCTc5o3VwOnhO23a2M1sLHbHLDywIz+n+y9cRcQ5vWUfvFZmIaopUUUskZjVkPNjbbR/hv8rYjZvaNxdKJQ+bD8aPly2pYhrHmfDufniqkDUNsL/nh6WlggagAO2PWws48cIYjpjvbHe+OYAPXwsMLfBhGOYAP/9k=" alt ="Breaking Bad" >
```

Comment:
Better use path to image, instead of embedding base64 encoded for better performance and less code. 
No space after attribute name: `alt="Breaking Bad"`